File: Algoritimos-fundamentais-Entregavel1/Contagem.java
import java.util.Scanner;

public class Contagem {
    public static void main(String[] args) {

        Scanner teclado = new Scanner(System.in);

        int n;
        int numero;
        int contador = 0;

        System.out.print("Digite N: ");
        n = teclado.nextInt();

        for (int i = 0; i < n; i++) {

            System.out.print("Digite um numero: ");
            numero = teclado.nextInt();

            if (numero >= 1 && numero <= n) {
                contador++;
            }
        }

        System.out.println("Quantidade de valores entre 1 e " + n + ": " + contador);

        teclado.close();
    }
}

File: Algoritimos-fundamentais-Entregavel1/Contagem.js
const readline = require("readline");

const teclado = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

teclado.question("Digite N: ", (entrada) => {

    let n = parseInt(entrada);
    let contador = 0;
    let i = 0;

    function pedirNumero() {

        if (i < n) {

            teclado.question("Digite um numero: ", (entrada) => {

                let numero = parseInt(entrada);

                if (numero >= 1 && numero <= n) {
                    contador++;
                }

                i++;
                pedirNumero();
            });

        } else {

            console.log("Quantidade de valores entre 1 e " + n + ": " + contador);

            teclado.close();
        }
    }

    pedirNumero();
});